packages/apify/src/internals/charging.ts

```diff
@@ -6,8 +6,6 @@ import { Configuration } from '../configuration.js';
 
 /**
  * Handles pay-per-event charging.
- *
- * @internal
```

Comment:
we'll need to regenerate the typedoc json for this to appear in the stable docs too